src/BenchmarksDriver/Program.cs

```diff
@@ -87,6 +87,10 @@ public static int Main(string[] args)
                 "Source dependency. Format is 'repo@branchOrCommit'. " +
                 "Repo can be a full URL, or a short name under https://github.com/aspnet.",
                 CommandOptionType.MultipleValue);
+            var webHostOption = app.Option(
```

Comment:
It looks like the short option `-w` is still free, so I would add it to `--webHost`.